### DISCUSSION TITLE
Live Grep and File Search Working Oddly

### DISCUSSION THREAD

**Discussion by alitinsky-ngt** [NvChad/NvChad#2225] [Q&A] [answered]
I installed ripgrep, and live grep and file search is working but in my config for example, it doesn't list any of the custom files or folders. The same applies when I live grep it as demonstrated in the video. What is the best way to fix this? 

https://github.com/NvChad/NvChad/assets/139795743/6ddb618e-e97e-46be-b6c1-f749da8e98b9

**Comment by siduck** [comment]
its respecting your .gitignore, there must be a setting to avoid this i'm sure, look up telescope docs :)

**Reply by antoineco** [reply]
Does it though? The files are showing in NvimTree. If they were gitignored, they would also be hidden there (unless explicitly toggled).

**Reply by alitinsky-ngt** [reply]
The custom folder is indeed in the .gitignore. I think this is what was causing the issue.

**Reply by alitinsky-ngt** [reply, marked as answer]
I used the command :Telescope find_files no_ignore=true and it started working.

**Comment by antoineco** [comment]
Live Grep searches **inside files** (all files in your current location, not only the one you selected in NvimTree).

The reason why you're not seeing any result is because none of the files under `lua/custom` contain the word "custom".
Try another word like "require" and you will see that results are returned inside those files.
